lib/brook/storage/redis.ex

```diff
@@ -49,82 +90,50 @@ defmodule Brook.Storage.Redis do
     redix_args = Keyword.fetch!(args, :redix_args)
     namespace = Keyword.fetch!(args, :namespace)
 
+    :ets.new(__MODULE__, [:set, :protected, :named_table])
+    :ets.insert(__MODULE__, {:namespace, namespace})
+
     {:ok, %{namespace: namespace}, {:continue, {:init, redix_args}}}
   end
 
   @impl GenServer
   def handle_continue({:init, redix_args}, state) do
     {:ok, pid} = Redix.start_link(redix_args)
+    :ets.insert(__MODULE__, {:redix, pid})
     {:noreply, Map.put(state, :redix, pid)}
   end
 
-  @impl GenServer
-  def handle_call({:persist, event, collection, key, value}, _from, state) do
-    Logger.debug(fn -> "#{__MODULE__}: persisting #{collection}:#{key}:#{inspect(value)} to redis" end)
-
-    Redix.command!(state.redix, ["SET", key(state, collection, key), :erlang.term_to_binary(%{key: key, value: value})])
-
-    Redix.command!(state.redix, [
-      "RPUSH",
-      events_key(state, collection, key),
-      :erlang.term_to_binary(event, compressed: 9)
-    ])
-
-    reply(:ok, state)
-  end
-
   @impl GenServer
```

Comment:
Did you mean to leave this lone `handle_call/3` from the old `:delete` implementation when you got rid of the others in favor of the `redis_*` functions?

Reply:
Good catch, I will delete it.